omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -3562,6 +3575,8 @@
 
                                                                             <obs conceptId="374" answerConceptId="5277"
                                                                                  answerLabel="Fertility Awareness Methods"/>
+                                                                            <obs conceptId="374" answerConceptId="175402"
```

Comment:
Is this part of the changes on the new tools?? I dont seem to see it anywhere on the old tools